src/test/kotlin/Utilities/FileIoTest.kt

```diff
@@ -43,4 +43,13 @@ class FileIoTest {
         val fileIo = FileIo()
         assertFailsWith<NullPointerException> { fileIo.readFile("static-no-file.csv") }
     }
+
+    @Test
+    fun `clear file returns an empty file`() {
+        val fileIo = FileIo()
+        val path = "src/test/kotlin/resources/clear-file.txt"
```

Comment:
The name of this file makes me think that it might already be empty. Does it contain any data before you clear it?